lib/internal/errors.js

```diff
@@ -64,13 +64,10 @@ class SystemError extends Error {
 
     super(message);
     Object.defineProperty(this, kInfo, {
-      configurable: false,
-      enumerable: false,
       value: context
     });
     Object.defineProperty(this, kCode, {
       configurable: true,
-      enumerable: false,
```

Comment:
Why these changes?

Reply:
I guess it's just me that I like the implicit version since I know very well that the default is false and it's less code.

I'll change it back to the way before.